src/frontend/src/helpers/Services.ts

```diff
@@ -106,12 +106,20 @@ namespace Services {
 
     // login successful
     await localStorage.connect(`seashell-${connection.username}`);
+    await localStorage.deleteDB();
+    await localStorage.connect(`seashell-${connection.username}`);
```

Comment:
like appending a Date.now() after? It will litter your browser with several hundreds megabytes of data